recipe/bld.bat

```diff
@@ -9,6 +9,9 @@ cl ^
     /DSQLITE_ENABLE_RTREE ^
     /DSQLITE_ENABLE_GEOPOLY ^
     /DSQLITE_ENABLE_COLUMN_METADATA=1 ^
+    /DSQLITE_MAX_VARIABLE_NUMBER=250000 ^
```

Comment:
Is this needed for the macOS and Linux builds?

Reply:
It's already there in unix builds.